calcom/cal.com
# fix: Default to Daily if Viable

## What does this PR do?



This PR fixes a bug where an event type would default to Daily even if it would throw an error.

Fixes # (issue)



## Requirement/Documentation



- If there is a requirement document, please, share it here.
- If there is ab UI/UX design document, please, share it here.

## Type of change




## How should this be tested?



- Create a new event type, do not set the location
- Create a booking under that new event type
    - The `EventManager` should not throw an error

## Mandatory Tasks


## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
